### DISCUSSION TITLE
Suggestion: 'Height Marks on Scenery'

### DISCUSSION THREAD

**Discussion by Nubbie** [OpenRCT2/OpenRCT2#15686] [Feature requests]
We have height marks in the 'View options' for:
Land, Ride tracks, Paths.

Why not for scenery? Great if you're going to do larger sceneris which is a pain if you have to measure distances and heights with your eyes. It's not in the way and more of a neat feature. 
Had the problem today when I tried to 'mirror' parts of a castle where it sometimes got a bit too high or too low

**Comment by DyXel** [comment]
It would be a great addition. Also, what about drawing lines that indicate where the scenary object really is?

For example, look at this:
![2016-04-25-004449_1024x768_scrot](https://cloud.githubusercontent.com/assets/11878637/14774388/80a4f528-0a7f-11e6-8dec-8fcd6878b2b4.png)
You really do not know in which square the object is (and also, is confusing).

The line would be something like this: (maybe a bit translucent and thinner)
![idea](https://cloud.githubusercontent.com/assets/11878637/14774420/d26c3dee-0a7f-11e6-8a79-ac30462b53ed.png)

**Comment by janisozaur** [comment]
I don't know if we can do that now, but after seeing a post on reddit (https://www.reddit.com/comments/4g5ytm) I support this feature as well.

**Comment by IntelOrca** [comment]
My alternative fix for that issue is allowing custom viewports to be individually rotated. We have to wait a bit though as currently everything shares the same rotation.

**Comment by X123M3-256** [comment]
Height marks on scenery would have to be restricted somewhat - if you consider how many scenery objects are in a typical park, the whole viewport would be plastered with height markers. I think it should just be the topmost item on the tile and the item currently being placed that have markers, or something like that.

**Comment by fromthere** [comment]
or Make an inspection tool that can get information(x,y,z,side(object like fence will need this)) of object. that would solve problem since all problem is not knowing where the objects are.
